src/utils/url.ts

```diff
@@ -1,7 +1,9 @@
 import { ASSETS_ROOT } from '../constants';
 
-export function wrapUtm(url?: string, prefix: string = '?') {
-  return `${url}${prefix}utm_medium=email&utm_source=email`;
+export function wrapUtm(url: string) {
+  return `${url}${
+    url.includes('?') ? '&' : '?'
+  }utm_medium=email&utm_source=email`;
```

Comment:
I think this should work (assume legit url), but not sure if we want to use a url parsing lib here instead

Reply:
Keep it simple first until we have to manipulate qs other than this function